### PR TITLE
taxonomy: rare legumes and wheat / divinfood project

### PR DESCRIPTION
**Pull request by stephanegigandet**
Some rare legumes and cereals that are of interest to the Divinfood EU project.

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
 #
 ##################################################################################
 
+# Triticum turgidum subsp. dicoccum and T. t. conv. durum
 <en:cereal
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22461]:
```suggestion
<en:wheat
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
...
-en:emmer
+en:emmer, emmer wheat
 de:Emmer
+es:trigo farro, farro
 fr:blé amidonnier, amidonnier
+it:farro dicocco
+la:Triticum dicoccum
+pt:farro
 ru:двузернянка
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22469]:
```suggestion
ru:двузернянка
wikidata:en:Q167339
```

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
 #
 ##################################################################################
 
+# Triticum turgidum subsp. dicoccum and T. t. conv. durum
 <en:cereal
-en:emmer
+en:emmer, emmer wheat
 de:Emmer
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22463]:
```suggestion
ar:قمح ثنائي الحبة
bg:Двузърнест лимец
ca:Pisana
cv:Пăри
da:Emmer
de:Emmer
```

File: taxonomies/ingredients.txt [hunk 4]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
...
+es:trigo farro, farro
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22464]:
```suggestion
el:Δίκοκκο σιτάρι
eo:Triticum turgidum subsp. dicoccon
es:trigo farro, farro, Triticum dicoccum
```

File: taxonomies/ingredients.txt [hunk 5]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
...
 fr:blé amidonnier, amidonnier
+it:farro dicocco
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22466]:
```suggestion
fa:گندم دودانه
fi:Emmervehnä
fr:Amidonnier
hi:खपली गेहूँ
hu:Tönkebúza
hy:Հաճար
it:farro dicocco, Triticum dicoccum
```

File: taxonomies/ingredients.txt [hunk 6]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
...
+la:Triticum dicoccum
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22467]:
```suggestion
ja:エンマーコムギ
ka:ასლი
la:Far, Triticum dicoccum
```

File: taxonomies/ingredients.txt [hunk 7]
```diff
@@ -22453,13 +22457,23 @@ fr:boulgour de blé dur
...
+pt:farro
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:22468]:
```suggestion
ml:എമ്മെർ
nl:Emmertarwe
nn:Emmer
os:Уасылы
pl:Pszenica płaskurka
pt:farro
```

File: taxonomies/ingredients.txt [hunk 8]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
 en:broad bean protein
 nl:tuinbooneiwit
 
+# rare bean from the Lyon region in France
+<en:legume
+en:meat bean
+fr:haricot viande, haricots viande
+
+<en:legume
+en:Adzuki bean, azuki bean
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54147]:
```suggestion
en:Adzuki bean, azuki bean
ar:لوبياء مقرنة
ca:Mongeta azuki
cs:Vigna hranatá
```

File: taxonomies/ingredients.txt [hunk 9]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+de:Adzukibohne
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54148]:
```suggestion
de:Adzukibohne
de:Adzukibohne
eo:Azukio
es:Vigna angularis
eu:Azuki
fa:لوبیای آزوکی
fi:Adsukipapu
```

File: taxonomies/ingredients.txt [hunk 10]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+fr:haricot azuki, haricots azuki
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54149]:
```suggestion
fr:haricot azuki, haricots azuki
he:אזוקי
hu:Azukibab
id:Kacang azuki
```

File: taxonomies/ingredients.txt [hunk 11]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+it:fagiolo azuki, fagioli azuki
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54150]:
```suggestion
it:fagiolo azuki, fagioli azuki, vigna angularis
ja:アズキ
ko:팥
```

File: taxonomies/ingredients.txt [hunk 12]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+la:vigna angularis
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54151]:
```suggestion
la:vigna angularis
lt:Japoninė pupuolė
ms:Kacang merah
```

File: taxonomies/ingredients.txt [hunk 13]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+nl:adukiboon
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54152]:
```suggestion
nl:adukiboon
pl:Fasola adzuki
pt:Vigna angularis
ru:Адзуки
sr:Азуки пасуљ
sv:Adzukiböna
sw:Muazuki
th:ถั่วอาซูกิ
tr:Vigna angularis
uk:Адзукі
vi:Đậu đỏ
za:Lwglimz
zh:紅豆
wikidata:en:Q380279
```

File: taxonomies/ingredients.txt [hunk 14]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+
+<en:legume
+en:cowpea, cowpeas
+de:Augenbohne
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54156]:
```suggestion
de:Augenbohne
dv:ރިހަ ތޮޅި
el:Αμπελοφάσουλο
eo:Okulvigno
```

File: taxonomies/ingredients.txt [hunk 15]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+es:feijão-frade
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54157]:
```suggestion
es:feijão-frade, vigna unguiculata
fa:لوبیا چشم‌بلبلی
fi:Lehmänpapu
```

File: taxonomies/ingredients.txt [hunk 16]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+fr:niébé, voème
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54158]:
```suggestion
fr:niébé, voème, vigna unguiculata
gu:ચોળા
he:לוביה
hi:लोबिया
is:Augnbaun
```

File: taxonomies/ingredients.txt [hunk 17]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+it:fagiolo dall'occhio
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54159]:
```suggestion
it:fagiolo dall'occhio, vigna unguiculata
ja:ササゲ
jv:Polong sapi
ka:ძაძა
ko:동부 (식물)
```

File: taxonomies/ingredients.txt [hunk 18]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+la:vigna unguiculata
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54160]:
```suggestion
la:vigna unguiculata, vigna unguiculata
mg:Voanemba
ml:വൻപയർ
ne:बोडी
pa:ਲੋਬੀਆ
pl:Wspięga wężowata
pt:Vigna unguiculata
ru:Коровий горох
rw:Igishyimbo
sg:Gbari
si:කව්පි
sn:Nyemba
sr:Крављи грашак
sv:Ögonböna
sw:Mkunde
ta:காராமணி
te:అలసంద
tl:Paayap
tr:Börülce
uk:Вігна китайська
vi:Đậu dải
zh:豇豆
wikidata:en:Q498940
```

File: taxonomies/ingredients.txt [hunk 19]
```diff
@@ -54124,6 +54138,27 @@ la:Vicia faba
...
+
 # description:en:The pea is most commonly the small spherical seed or the seed-pod of the pod fruit Pisum sativum. Fresh peas are often eaten boiled and flavored with butter and/or spearmint as a side dish vegetable. Peas are usually boiled or steamed, which breaks down the cell walls and makes the taste sweeter and the nutrients more bioavailable.
 # comment:en:The wikidata translations are not clear between seed name, plant name and latin name.
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54163]: I just sew that it after writing last few comments :-(

File: taxonomies/ingredients.txt [hunk 20]
```diff
@@ -54653,6 +54688,17 @@ en:grey pea, grey peas
...
+es:almorta, chicharo
+fi:peltonätkelmä
+fr:pois carré, gesse
+it:cicerchia
+la:lathyrus sativus
+pt:chícharo
+sv:plattvial
+
 # description:en:The lentil (Lens culinaris or Lens esculenta) is an edible legume. Lentils are consumed in many ways. They can be eaten soaked, germinated, boiled, fried and baked.
 # comment:en:The wikidata entry is more about the plan, so translations can be wrong.
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:54703]:
```suggestion
# comment:en:The wikidata entry (Q108363340) is more about the plan, so translations can be wrong.
```

File: taxonomies/ingredients.txt (hunk 6 above)
Comment (stephanegigandet) [reply taxonomies/ingredients.txt:22467]: is the la:Far, addition intended?

File: taxonomies/ingredients.txt (hunk 9 above)
Comment (stephanegigandet) [reply taxonomies/ingredients.txt:54148]: there is a duplicate line for de:

File: taxonomies/ingredients.txt (hunk 18 above)
Comment (stephanegigandet) [reply taxonomies/ingredients.txt:54160]: for la: there is the same entry twice?

**Comment by stephanegigandet** [comment]
@benbenben2 Thanks a lot for all the additions! For some reason GitHub does not let me apply them, are you able to commit them?

**Comment by benbenben2** [comment]
> @benbenben2 Thanks a lot for all the additions! For some reason GitHub does not let me apply them, are you able to commit them?

Same here, I have a "This diff has recently been updated. Refresh and try again." message when I try to apply changes. And refresh and update branch could not change anything.

**Comment by stephanegigandet** [comment]
@benbenben2 I copied your suggestions manually. I removed the translations that are just latin, as it's not the usual name we find on food products, it's more of a wikipedia convention. In practice we do match Latin synonyms for ingredients in all languages already.

**Review by benbenben2 — APPROVED** [review]

**Pull request merged**